server.go

```diff
@@ -540,6 +540,20 @@ func (s *Server) errorf(format string, a ...interface{}) {
 	}
 }
 
+// ServiceRegistrar wraps a single method that supports service registration. It
+// enables users to pass concrete types other than grpc.Server to the service
+// registration methods exported by the IDL generated code.
+//
+// This API is EXPERIMENTAL.
+type ServiceRegistrar interface {
+	// RegisterService registers a service and its implementation to the
+	// concrete type implementing this interface. It is called from the IDL
+	// generated code.
```

Comment:
```go
// RegisterService registers a service and its implementation to the
// concrete type implementing this interface.   It may not be called
// once the server has started serving.
```

(I think we can omit the bit about generated code here.)

Reply:
Done.